src/main/java/frc/robot/commands/AssistedBalanceCommand.java

```diff
@@ -43,10 +44,10 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-
+    SmartDashboard.putNumber("PIDOut", -pidController.calculate(swerve_subsystem.getPitch(), 0.0));
```

Comment:
Do we want to keep this? If so, lets make a variable to store the calculate result so we don't have to do it twice. This version is also missing the MathUtil.clamp